security/pfSense-pkg-sudo/files/usr/local/pkg/sudo.inc

```diff
@@ -126,6 +127,15 @@ function sudo_get_users($list_all_user = false) {
 	$tmpuser["name"] = "user:root";
 	$tmpuser["descr"] = "User: root";
 	$users[] = $tmpuser;
+	
+	/* Make an entry for nagios, if the nrpe package is installed */
+        $nrpePackage = get_pkg_info('net-mgmt/pfSense-pkg-nrpe', true, true);
+        if (!empty($nrpePackage)) {
+                $tmpuser = array();
+                $tmpuser["name"] = "user:nagios";
+                $tmpuser["descr"] = "User: nagios";
+                $users[] = $tmpuser;
+        }
```

Comment:
Please use hard tabs to indent instead of spaces

Reply:
Apologies - change applied